### PR TITLE
Support for parametrized multilanguage testing

### LINKED ISSUE #1589: [Tests idea] Make Multilanguage tester
**Language idea incl level**
Right now the tests for the Hedy translator are hardcoded in each language. So, for example, if we want to test a feature there we'd call the test something like: `test_is_spanish_english` and it would include Spanish keywords.

I see 2 problems with this approach: only people who can speak the language can write the tests or understand it at first glance (I can understand Dutch tests, but I have to go to the grammar and see each definition),  and also, more importantly, we can only test one language at a time. 

Maybe we can do something like the Multilevel test, but for languages? We'd build each test so it take a generic placeholder like `lang['nl']['is]` and iterate through a language dictionary that contains all keywords for each language. Or perhaps this is not needed? Since the keywords for each language shouldn't affect the grammar and a test in a single language is all that is needed.

I'd love to read your input!

### PR DESCRIPTION
**Pull request by racertop**
**Description**

This PR adds a way to use parametrized tests for the Hedy translator. For this I added:

- A dictionary with the languages we support.
- A function to feed the keywords and languages to the tester.

**Fix for**

Fixes #1589 

**How to test**

Describe how to run your code to see it is action. See this https://github.com/Felienne/hedy/pull/880#issue-1016304308 for an example.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] Links to an existing issue or discussion (if not, create an issue first)
- [ ] Describes changes clear in the format above (present tense, no subject)
- [ ] Has a "how to test" section
- [ ] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)

### CODE REVIEW

File: tests/test_translation_level_02.py [hunk 1]
```diff
@@ -11,6 +13,7 @@
 
 class TestsTranslationLevel2(HedyTester):
     level = 2
+    all_kwords = hedy_translation.all_keywords_to_dict()
```
Comment (Felienne) [inline tests/test_translation_level_02.py:16]: Thanks for this work @racertop it is a great way to test more extensively!!

I would just like to ask, can we make this var name `all_keywords`? It is just 2 more letters and it is a lot more clear! In general we try not to use abbreviations unless they really save letters and are wellknown (like AST for abstract syntax tree)

**Review by Felienne — CHANGES_REQUESTED** [review]

File: tests/test_translation_level_02.py (hunk 1 above)
Comment (racertop) [reply tests/test_translation_level_02.py:16]: Ok! Do you want me to change it in the places where I use `print_kword` or `if_kword` too?

Comment (Felienne) [reply tests/test_translation_level_02.py:16]: Yeah that would be nicer!

**Comment by racertop** [comment]
The test is failing because the translation Dutch translation differs in a keyword with the Spanish and English version. Particulary the command `add` in English and Spanish differs from `add_list` in Dutch. I assumed those dictionaries would all have the same key.

**Comment by Felienne** [comment]
Ow good catch! I updated that recently in English but I forgot to update
the others. Can you make it all the same as en?

**Comment by racertop** [comment]
> Ow good catch! I updated that recently in English but I forgot to update the others. Can you make it all the same as en?

Done! I changed all occurences of kword to keyword as well. If there's anything else to change, I will fix that! 

Thank you for your time!

**Review by Felienne — APPROVED** [review]

**Pull request merged**